.github/workflows/shellcheck.ignore.yaml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
     paths-ignore:
-      - "**/*.sh"
+      - "**.sh"
```

Comment:
weird syntax given that '*' usually means match anything/everything/nothing...

Reply:
Yep. It is weird